Game/Source/GDKTestGyms/BenchmarkGymGameModeBase.cpp

```diff
@@ -419,8 +427,7 @@ void ABenchmarkGymGameModeBase::Tick(float DeltaSeconds)
 	
 	// PrintMetricsTimer needs to be reset at the the end of ABenchmarkGymGameModeBase::Tick.
 	// This is so that the above function have a chance to run logic dependant on PrintMetricsTimer.HasTimerGoneOff().
-	if (HasAuthority() &&
-		PrintMetricsTimer.HasTimerGoneOff())
+	if (PrintMetricsTimer.HasTimerGoneOff())
```

Comment:
Why this change?

Reply:
This runs on non-auth servers now. Part of the "allow UX reporting to work in zoning scenarios commit".